cfgov/cfgov/settings/base.py

```diff
@@ -177,7 +177,8 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.environ.get('DJANGO_STATIC_ROOT', '/var/www/html/static')
 
-MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'f')
+MEDIA_ROOT = os.environ.get('MEDIA_ROOT', 
+                            os.path.join(PROJECT_ROOT, 'f'))
```

Comment:
That said, I don't think this should block this particular PR. We should have this conversation though.